package.json

```diff
@@ -202,7 +202,7 @@
     "prettier": "^3.2.5",
     "sass": "^1.74.1",
     "supertest": "^7.0.0",
-    "ts-jest": "^29.1.2",
+    "ts-jest": "~29.1.2",
```

Comment:
Are we confident about future minor patches on this?

Reply:
As discussed, it seems I might have misinterpreted the statement in the Versioning section in https://www.npmjs.com/package/ts-jest about them not following semver. What they mean by that might be that they might release minor versions without new features and major ones without breaking changes in order to stay aligned version-wise with jest. However, since the change here has been to a more restrictive pattern, and we should be checking for breaking changes whenever libraries are updated anyway, we've agreed to go ahead with this as is.